app/views/account/users/_user.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td= link_to user.email, user.email
+  td= user.name
+  td
+    - if current_user.admin?
+      = simple_form_for(user, url: account_user_path(user)) do |f|
```

Comment:
Agree. Will try to implement ajax update in next PR.

If we'll decide to edit something is user profile, "edit user form" will be also implemented